otel4s/src/test/scala/com/commercetools/queue/otel4s/MeasuringPullerSuite.scala

```diff
@@ -18,8 +18,9 @@ package com.commercetools.queue.otel4s
 
 import cats.data.Chain
 import cats.effect.IO
+import cats.implicits.toFoldableOps
```

Comment:
This is a legacy import. Prefer this one:

```suggestion
import cats.syntax.foldable._
```